modules/rostests/kmtests/ntos_cc/CcMapData_drv.c

```diff
@@ -10,6 +10,17 @@
 #define NDEBUG
 #include <debug.h>
 
+// Copied from 'ntoskrnl/include/internal/*/mm.h'.
+#ifdef _X86_
+    #define MI_SYSTEM_CACHE_START  (PVOID)0xC1000000
+    #define MI_PAGED_POOL_START    (PVOID)0xE1000000
+#elif defined(_M_AMD64)
+    #define MI_SYSTEM_CACHE_START         0xFFFFF98000000000ULL
+    #define MI_SYSTEM_CACHE_END           0xFFFFFA7FFFFFFFFFULL
+#else
+    // Related check will be skipped.
+#endif
+
```

Comment:
Why changing everything instead of just fixing what bothers you?

Reply:
Well, magic values do bother me (searching & reading)...

Reply:
Also this fixes x64!